ReactwithASP.NETCore/ClientApp/src/components/FetchData.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+
+export class FetchData extends Component {
+  static displayName = FetchData.name;
+
+  constructor(props) {
+    super(props);
+    this.state = { forecasts: [], loading: true };
+  }
+
+  componentDidMount() {
+    this.populateWeatherData();
+  }
+
+  static renderForecastsTable(forecasts) {
```

Comment:
Is these default methods needed?

Reply:
This component folder is not needed, so removed this file for my end.